src/main/java/lotto/view/ResultView.java

```diff
@@ -38,4 +33,17 @@ public static void printLottoResult(LottoMatchResult matchResult, ProfitRate pro
     System.out.printf("총 수익률은 %.2f입니다.(기준이 1이기 때문에 결과적으로 %s라는 의미임)%n",
         rate, status);
   }
+
+  private static void printRank(LottoMatchResult matchResult, LottoRank rank) {
+    if (rank == LottoRank.MISS) {
+      return;
+    }
+    int count = matchResult.countMatches(rank);
+    int prize = rank.prize();
+    if (rank == LottoRank.SECOND) {
```

Comment:
```suggestion
    if (rank.isSecond()) {
```
enum에도 메시지를 보낼 수 있음.